partition/bootloader_test.go

```diff
@@ -157,3 +161,23 @@ func (s *PartitionTestSuite) TestInstallBootloaderConfig(c *C) {
 		c.Assert(osutil.FileExists(fn), Equals, true)
 	}
 }
+
+func (s *PartitionTestSuite) TestRebootNoError(c *C) {
```

Comment:
can we test the argument to shutdown as well here ? we should have examples how to do that